docs/blog/2023-12-15-best-practices.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Best practices for best results with Groundlight
```

Comment:
It sounds kinda weird to have two "best"s ...

Maybe "Best practices for Groundlight detectors" or "Getting the most out of your Groundlight detectors"?